### ISSUE TITLE
Oversized Passbolt logo on healthcheck page

### ISSUE THREAD

**Issue by TheReptile** [passbolt/passbolt_api#432]
* Passbolt Version: 3.6.0.
* Platform and Target:
-- Operating system: Ubuntu 20.04.04
-- PHP: 7.4
-- Web server: Nginx 1.18.0
-- Database server: MariaDB 10.3.34

### What you did
Visit the healthcheck page. https://<site>/healthcheck

### What happened
On the right side is a Passbolt help logo, but the dimensions are completely off, I guess somewhere a typo was made.
From the html source:
```
<span class="svg-icon life-ring icon-only">
    <svg width="1792" height="1792" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
        <path d="M256 504c136.967 0 248-111.033 248-248S392.967 8 256 8 8 119.033 8 256s111.033 248 248 248zm-103.398-76.72l53.411-53.411c31.806 13.506 68.128 13.522 99.974 0l53.411 53.411c-63.217 38.319-143.579 38.319-206.796 0zM336 256c0 44.112-35.888 80-80 80s-80-35.888-80-80 35.888-80 80-80 80 35.888 80 80zm91.28 103.398l-53.411-53.411c13.505-31.806 13.522-68.128 0-99.974l53.411-53.411c38.319 63.217 38.319 143.579 0 206.796zM359.397 84.72l-53.411 53.411c-31.806-13.505-68.128-13.522-99.973 0L152.602 84.72c63.217-38.319 143.579-38.319 206.795 0zM84.72 152.602l53.411 53.411c-13.506 31.806-13.522 68.128 0 99.974L84.72 359.398c-38.319-63.217-38.319-143.579 0-206.796z"></path>
    </svg>
</span>
```

### What you expected to happen
I would expect to see a normal sized logo. 
Checked this in Firefox and Chrome.

If the width and height are set to 192 it looks normal.

**Comment by cedricalfonsi** [comment]
Thank you for the report @TheReptile.
I created a bug on our backlog ref PB-16666

**Comment by imrejonk** [comment]
I encountered the same bug after upgrading to Passbolt Pro 3.6.0, except that not the Passbolt logo but the lifebuoy icon appears oversized:
![Screenshot from 2022-06-07 13-27-44](https://user-images.githubusercontent.com/48715395/172368918-f57238c1-2cb9-436a-9250-abb82677bcf5.png)
Clearing the browser cache does not make a difference. There is also a "login" button in the right-hand corner, which does not make sense as I am already logged in.

The page also looks funny when logged out:
![Screenshot from 2022-06-07 13-33-32](https://user-images.githubusercontent.com/48715395/172369644-9d2a8d0a-800c-4ad5-92bb-acc20772c87b.png)

Furthermore, I noticed that the `/app/healthcheck` path no longer works, I had to use the `/healthcheck` path. Not sure if this is intended behavior.
